copter/source/docs/traditional-helicopter-first-flight-tests.rst

```diff
@@ -0,0 +1,37 @@
+.. _traditional-helicopter-first-flight-tests:
+
+==================
+First Flight Tests
+==================
+
+
+First Spool Up
+==============
+
+Check the CG of helicopter. It should be exactly on the point of the main shaft. If not, adjust battery until it is.
+
+For the first spool up, before trying to lift into hover, you should verify that the main rotor blades are tracking reasonably well. If not, excessive vibration will be created, perhaps high enough to cause attitude control errors or even cause an :ref:`ekf-inav-failsafe`.
+
+- Arm the vehicle. If there is a pre-arm failure, resolve it before proceeding. Usually you will need to wait until the GPS obtains lock and the EKF sets its origin as indicated by the ground control station messages (and, of course, the failure to arm!)
+- Before the automatic disarming timing elapses, engage the Motor Interlock switch and/or raise the collective to mid-stick (if using passthru or throttle curve RSC modes), which should be 0 degrees main blade pitch. The rotor should spool up to speed.
```

Comment:
"mid-stick"  you mean the throttle stick to mid position?  The reason I ask is that mid-stick may mean different collective blade pitch depending on whether the flight mode is stabilize or acro, especially in an aerobatic setup.   So for your aerobatic setup in acro, mid-throttle stick may mean 0 deg but for my heli in stabilize mode, mid-throttle stick is hovering.  You need to word this to be more universal.  

But there could be a gotcha, with your recommendation even when worded properly.  You are suggesting they raise the collective blade pitch above the H_MID_COL value to keep the vehicle from disarming.  That could be problematic with my new integrator scheme because it will release the integrator and if  you are on a slope  it will cause the integrator to grow and cause the aircraft to roll over.

I would say they need to just enable motor interlock before the disarming timer elapses.  I would NOT suggest raising the collective to fool the system.

Reply:
how about "raise the collective until Motor Interlock engages if no switch is used. Collective should be at or slightly below 0 degrees main blade pitch"

Reply:
I have never used interlock without a switch. How do you set that up?  Why would you want interlock to operate off of the collective stick position (TX throttle stick)?  That doesn’t sound like a good idea.

Reply:
for mode=passthru you are using output 8 to passthru RCin8 to control throttle via some kind of TX curve off the throttle stick which also controls collective, since Motor Interlock ("switch") is also RC8 by default, it must be above 1200us to activate interlock.... 
I have never used this mode in AP, just going by how the docs say it works....I do need to eliminate the part including internal throttle curve, since that still needs an RC8 switch...

Reply:
fixed